lib/src/constants/command_constants.dart

```diff
@@ -1,4 +1,5 @@
 /// Stores all the commands used throughout the app that
+library;
```

Comment:
Why is the library keyword added to this file, and the same for the one below?

Reply:
Se my other PR for Offline mode, I've answered there